edalize/generic.py

```diff
@@ -0,0 +1,270 @@
+# Copyright edalize contributors
+# Licensed under the 2-Clause BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-2-Clause
+
+import logging
+import os.path
+import platform
+import re
+import subprocess
+
+from edalize.edatool import Edatool
+from edalize.yosys import Yosys
+from importlib import import_module
+
+logger = logging.getLogger(__name__)
+
+""" Generic backend
+
+A core (usually the system core) can add the following files:
+
+- Standard design sources (Verilog only)
+
+- Constraints: unmanaged constraints with file_type SDC, pin_constraints with file_type PCF and placement constraints with file_type xdc
+
+"""
+
+
+class Generic(Edatool):
```

Comment:
This probably shouldn't be called Generic?  What tool flow is it executing?  And I think @olofk has got a claim on the Generic name already:  https://github.com/olofk/edalize/compare/master...generic_tool  (if only he'd merge it so it can be used ;-) )

Reply:
actually, I haven't realized that this file was here... as you see it's a cross-people contribution and I did not have everything under control :D sorry, removing it